packages/devtools_app/integration_test/test/live_connection/network_screen_test.dart

```diff
@@ -95,8 +97,26 @@ final class _NetworkScreenHelper {
     );
   }
 
-  Future<void> triggerRequest(String path) async {
-    await http.get(Uri.parse('http://localhost:$_controlPort/$path'));
+  Future<void> triggerExit() async {
+    final response = await serviceConnection.serviceManager
+        .callServiceExtensionOnMainIsolate('ext.networking_app.exit');
+    logStatus(response.toString());
+
+    await Future.delayed(const Duration(milliseconds: 200));
```

Comment:
What is the `delayed` here for? (I'm wondering if there is a future we could explicitly wait on instead...)